src/sentry/conf/server.py

```diff
@@ -2930,5 +2931,10 @@ def build_cdc_postgres_init_db_volume(settings):
 # decorator.
 SINGLE_SERVER_SILO_MODE = False
 
+# Used to determine if we should or not record an analytic data for a first event of a project with a minified stack trace
+START_DATE_TRACKING_FIRST_EVENT_WITH_MINIFIED_STACK_TRACE_PER_PROJ = datetime(
+    2022, 12, 14, tzinfo=pytz.UTC
+)
```

Comment:
Doesn't seem to belong here.
In settings you want to have things that can be configured by users to suit the installation... when would a user want to change this, if never it's probably doesn't belong here ?

Reply:
I agree that since this is only being used in one place and we don't want it to be configurable, it doesn't make sense to be here